docs/kubernetes/README.md

```diff
@@ -189,8 +189,26 @@ $ kubectl apply -f https://raw.githubusercontent.com/hetznercloud/csi-driver/v2.
 
 ## Integration with Root Servers
 
-Root servers can be part of the cluster, but the CSI plugin doesn't work there. Taint the root server as follows to skip that node for the DaemonSet.
+Root servers can be part of the cluster, but the CSI plugin doesn't work there and the current behaviour of the scheduler can cause Pods to be stuck in `Pending`. 
 
+Include topology key evaluation into storage class, by setting the helm chart value `includesRobotServers`:
+
+```yaml
+storageClasses:
+  - name: hcloud-volumes
+    defaultStorageClass: true
+    reclaimPolicy: Delete
+    includesRobotServers: true # <---
+```
+
+Label the nodes according to their type:
+
+**Cloud Servers**
+```bash
+kubectl label nodes <node name> instance.hetzner.cloud/is-root-server=false
```

Comment:
Considering that we have users with clusters with even more mixed servers (Cloud, Robot, Raspberry Pis at home, ...) I think this nomenclature is confusing.

What do you think about adding a new annotation `instance.hetzner.cloud/{source,type,service,owner}: {cloud,robot}` that we can set in HCCM and use here?